types/mixpanel-browser/index.d.ts

```diff
@@ -169,7 +169,7 @@ export interface Mixpanel {
     ): void;
     track_forms(query: Query, event_name: string, properties?: Dict | (() => void)): void;
     track_links(query: Query, event_name: string, properties?: Dict | (() => void)): void;
-    track_pageview(properties?: Dict): void;
+    track_pageview(properties?: Dict, options?: Dict): void;
```

Comment:
Shouldn't this be 
```suggestion
    track_pageview(properties?: Dict, options?: { event_name?: string | undefined }): void;
```
according to docs: https://github.com/mixpanel/mixpanel-js/blob/master/doc/readme.io/javascript-full-api-reference.md#mixpaneltrack_pageview?